pkg/api/server/register_images.go

```diff
@@ -540,6 +540,13 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//    description: |
 	//      Suppress verbose build output
 	//  - in: query
+	//    name: compatvolumes
+	//    type: boolean
+	//    default: false
+	//    description: |
+	//      Contents of base images to be modified on ADD or COPY only
+	//      (As of version 1.37)
```

Comment:
nit: readers have no idea what this version is when read podman API docs, so this should really say podman 5.2 or just leave the version out.
But not worth to repush.

Reply:
I'll clean up after this merges.